src/chrome/content/rules/yplanapp.com.xml

```diff
@@ -1,11 +1,12 @@
 <ruleset name="YPlan">
 
-	<target host="yplanapp.com" />
-	<target host="www.yplanapp.com" />
+	<target host="*.yplanapp.com" />
 
-  <securecookie host="^(www\.)?yplanapp.com$" name=".+" />
+	<securecookie host="^.*\.yplanapp\.com$" name=".+" />
```

Comment:
I got the example from` 	<securecookie host="^.*\.123-reg\.co\.uk$" name=".*" />` is that wrong too?